src/components/Tooltip/index.test.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
-import { mount } from 'enzyme';
-
+import { render, waitFor, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import userEvent from '@testing-library/user-event';
 import Tooltip from '.';
 
-const Trigger = () => <p>trigger</p>;
-
 describe('Tooltip component', () => {
-  it('matches snapshot (uses the right classes)', () => {
-    const wrapper = mount(<Tooltip trigger={<Trigger />} message="lol" />);
-    expect(wrapper).toMatchSnapshot();
+  it('tests that message gets rendered on hover', async () => {
+    render(<Tooltip trigger={<button>trigger</button>} message="lol" />);
+    userEvent.hover(screen.getByText('trigger'));
+    await waitFor(() => {
+      expect(screen.getByText('lol')).toBeTruthy();
+    });
```

Comment:
The importance of creating tests like this cannot be understated. One thing to note, is that in the snapshot test, the message of "lol" should NOT be rendering as it hasn't been hovered. Moving this to a testing-library test not only makes this more painfully obvious, it makes it easy to add a hover event to properly trigger it. This is unlikely to be a false positive / false negative moving forward.

Reply:
The old test and the new test are testing different things (class name application vs if it gets rendered at all) - but point taken, yours is a more useful test